src/App.tsx

```diff
@@ -1,24 +1,195 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { UserWarning } from './UserWarning';
+import { Form } from './components/Form';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import { Footer } from './components/Footer';
+import { Status } from './types/Status';
+import { getTodos, updateTodo, deleteTodo } from './api/todos';
 
-const USER_ID = 0;
+const USER_ID = 12132;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filtredByStatus, setFiltredByStatus] = useState<Status>(Status.ALL);
+  const [errorNotification, setErrorNotification] = useState<string | null>(
+    null,
+  );
+  const [errorVisible, setErrorVisible] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [procesingTodoIds, setProcesingTodoIds] = useState<number[]>([]);
+
+  const showErrorNotification = (message: string) => {
+    setErrorNotification(message);
+    setErrorVisible(true);
+
+    setTimeout(() => {
+      setErrorVisible(false);
+      setErrorNotification(null);
+    }, 3000);
+  };
+
+  useEffect(() => {
+    getTodos(USER_ID)
+      .then((data) => {
+        setTodos(data);
+      })
+      .catch(() => {
+        showErrorNotification('Unable to load todos');
```

Comment:
I think this should be in some enum ErrorType, for example:
```
export enum ErrorType {
  load = 'Unable to load todos',
  title = 'Title should not be empty',
  ...
  }
  ```
  and then use 
  ```
  showErrorNotification(ErrorType.load);
  ```

Reply:
true